.circleci/config.yml

```diff
@@ -1,6 +1,13 @@
 version: 2.1
 
 commands:
+  install-xcpretty:
+    steps:
+      - run:
+          name: Install xcpretty
+          command: |
+            gem install --user-install xcpretty
+            echo 'export PATH="$HOME/.gem/ruby/2.3.0/bin:$PATH"' >> $BASH_ENV
```

Comment:
Never seen this way of adding a path to the env, does this work?

Reply:
Yup that's the way CircleCI does env variables :/